pkg/ddevapp/drupal.go

```diff
@@ -593,3 +593,31 @@ func drupalImportFilesAction(app *DdevApp, uploadDir, importPath, extPath string
 
 	return nil
 }
+
+// getDrupalComposerCreateAllowedPaths returns paths that are allowed to be present when running composer create
+func getDrupalComposerCreateAllowedPaths(app *DdevApp) ([]string, error) {
```

Comment:
I have to confess that this looks pretty complex compared to what we expect it to need to be.

Reply:
is there something to do here?

Reply:
Take a look and see if it has to be this much code to accomplish a list of paths.

Reply:
I simplified it slightly moving some of what it does upwards.